README.md

```diff
@@ -10,6 +10,10 @@ For more information, [visit our website](https://musicbrainz.org/doc/About).
 
 To get help, please join #metabrainz on irc.freenode.net
 
+Important announcements (e.g. about breaking changes to the schema or our 
+API / web service) will be made on [our blog](https://blog.metabrainz.org/),
```

Comment:
- +database _schema_
- -parenthesis
- replace _important announcements_ with _breaking changes_ as the link specifically targets that in next commit.

```suggestion
Breaking changes to the database schema or our API / web service will be announced on [our blog](https://blog.metabrainz.org/),
```